Lib/test/test_capi/test_codecs.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+from test import support
+
+try:
+    import _testcapi
+except ImportError:
+    _testcapi = None
+
+
+class CAPITest(unittest.TestCase):
+
+    @support.cpython_only
```

Comment:
For me it's redundant with testing _testcapi. For example, PyPy doesn't have _testcapi. But you can keep it if you prefer to be explicit.